api/db/changelog/db.changelog-175-add-ct-training-access-module.xml

```diff
@@ -0,0 +1,25 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<databaseChangeLog
+  xmlns="http://www.liquibase.org/xml/ns/dbchangelog"
+  xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+  xsi:schemaLocation="http://www.liquibase.org/xml/ns/dbchangelog
+                      http://www.liquibase.org/xml/ns/dbchangelog/dbchangelog-3.1.xsd">
+    <changeSet author="yonghao" id="changelog-175-add-ct-training-access-module">
+        <sql>
+            ALTER TABLE access_module
+            MODIFY COLUMN name
+            ENUM('ERA_COMMONS',
+            'TWO_FACTOR_AUTH',
+            'RAS_LOGIN_GOV',
+            'RT_COMPLIANCE_TRAINING',
+            'DATA_USER_CODE_OF_CONDUCT',
+            'PROFILE_CONFIRMATION',
+            'PUBLICATION_CONFIRMATION',
+            'CT_COMPLIANCE_TRAINING')
+        </sql>
+        <sql>
+            INSERT INTO access_module (name, expirable, bypassable)
+            VALUES('CT_COMPLIANCE_TRAINING', true, true)
```

Comment:
Making this expirable=true implies that it's covered by AAR, which we are not planning to do for initial launch.  Let's make this false so we don't have to deal with the added complexity.